content/docs/command-reference/run.md

```diff
@@ -106,10 +106,10 @@ Relevant notes:
   also means that the stage command needs to recreate any directory structures
   defined as outputs every time its executed by DVC.
 
-- In some situations we have executed a stage and later notice that some of the
-  output files or directories it creates, which are already in the workspace,
-  are missing from `dvc.yaml` (`outs` field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+- In some situations, we have previously executed a stage, and later notice that
```

Comment:
situations , ... , and later , ... that something/something used by, or .... 

and only after that 

are missing 

this is very hard to read still. to many clauses per sentence.

in most cases in this PR it should be significantly simplified still.

Reply:
It should be easier to read after it gets generalized per https://github.com/iterative/dvc.org/pull/1914#discussion_r528391826 above.

Reply:
Just rewrote this and did other updates to how-tos per some recent discussions around `remove` on Discord. PTAL.